SConstruct

```diff
@@ -44,13 +45,37 @@ def set_defaults(env):
 
 def preload_prereqs(prereqs):
     """Preload prereqs specific to platform"""
+    prereqs.define('mchecksum',
+                   retriever=GitRepoRetriever("https://github.com/mercury-hpc/"
+                                              "mchecksum.git"),
+                   commands=['cmake -DBUILD_SHARED_LIBS=ON $MCHECKSUM_SRC '
```

Comment:
so is isal insufficient in that it doesn't provide a checksum type we want?

Reply:
mchecksum provides different implementations of the same checksums. I was only including it to compare the performance. I think I will remove it completely now.